avrogen/src/main/resources/ObjC/enumObjC.h.vm

```diff
@@ -22,3 +22,9 @@ typedef enum {
     ${ENUM_NAME}_$symbol,
 #end
 } $namespacePrefix${enum_name};
+
+# define ${namespacePrefix}${enum_name}Array(enum) [@[ \
+#foreach ($symbol in $schema.getEnumSymbols())
+@"${ENUM_NAME}_$symbol", \
```

Comment:
Enums in Objective-C aren't assigned to any package thus their names must be unique across the whole application.